hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/HapiPropertySource.java

```diff
@@ -225,6 +225,12 @@ static AccountID asAccount(String v) {
                 .build();
     }
 
+    static AccountID asAliasAccount(String v) {
+        return AccountID.newBuilder()
+                .setAlias(ByteString.copyFrom(v.getBytes()))
```

Comment:
I don't get it.  `String.getBytes` doesn't take the hex value and turn it into a number.  It just takes the characters of the string.  So that must mean that having an alias that consists not of bytes but of the hex-encoded bytes, since what is passed here (in the test below) is a string that looks like `0x0000000000000000000000691752902764108185`.  Am I understanding this right?

Reply:
well ... maybe it's just me ... but maybe a _different_ way of writing an invalid alias such that it doesn't _look_ like you're trying hard to do a hex-encoding->binary conversion?

Reply:
ok.  then, final suggestion if you'd like: Put a comment somewhere to say what kind of misuse of bytes is going on here so that the alias is invalid?  Or whatever is actually happening here?

Reply:
I mean, now I'm confused as to _why_ this alias is invalid.  Is it becauase it is an ascii string representing a hex encoding, or is that the way it is done and the invalid part is that the alias specified `0x0000000000000000000000691752902764108185` doesn't exist?

(If this alias field is _supposed_ to be the ascii string which is the hex-encoding of something, where is _that_ specified? I mean, it doesn't seem to be the base32url encoding specified by HIP-32, so what is it?)